web/experiments/gaussian_embedding.py

```diff
@@ -0,0 +1,79 @@
+import datetime
```

Comment:
As with the script, I think this is not a good fit for the package

Reply:
Why?

Reply:
I do not think it will be very useful for the users ;) This just follows directly from the principle "less is more". We can keep it on your fork for instance.